torch/csrc/distributed/rpc/ProcessGroupAgent.cpp

```diff
@@ -53,17 +53,66 @@ Message deserialize(std::istream& is) {
   return Message(std::move(payload), std::move(tensors), type, id);
 }
 
+std::unordered_map<std::string, int> collectNames(
```

Comment:
Why don't we make this a private member of ProcessGroupAgent since we need pg?

Reply:
Will make the change.